test/util/ChartUtils/getStackedDataOfItem.spec.ts

```diff
@@ -42,31 +56,49 @@ describe('getStackedDataOfItem', () => {
     const item1 = makeItem<string>('a');
     const item2 = makeItem<string>('a');
     const group = Symbol('my mock group');
-    const stackGroups = { a: { stackedData: [group], items: [item1] } };
+    const stackGroups: Record<StackId, GenericChildStackGroup<typeof group>> = {
+      a: {
+        stackedData: [group],
+        items: [item1],
+        numericAxisId: '',
+        cateAxisId: '',
+      },
+    };
     const result = getStackedDataOfItem(item2, stackGroups);
     expect(result).toBe(null);
   });
 
   it('should return stackedData if stack group contains the item matched by string', () => {
     const item = makeItem<string>('a');
     const group = Symbol('my mock group');
-    const stackGroups = { a: { stackedData: [group], items: [item] } };
+    const stackGroups: Record<StackId, GenericChildStackGroup<typeof group>> = {
+      a: {
+        stackedData: [group],
+        items: [item],
+        numericAxisId: '',
+        cateAxisId: '',
+      },
+    };
```

Comment:
How about extracting a helper function to generate the stackGroup? 

```
generateMockStackGroupFor(group, item){
    return {
      a: {
        stackedData: [group],
        items: [item],
        numericAxisId: '',
        cateAxisId: '',
      },
    };
}
```

Reply:
Added